cloud_functions/token_refresh/main.py

```
@@ -1,314 +1,314 @@
+        )
+        return https_fn.Response(
+            status=401,
+            response=json.dumps(
+                {"error": "Unauthorized", "message": "Unauthorized - Revoked token"}
+            ),
+            headers={
+                "Content-Type": "application/json",
+                "Access-Control-Allow-Origin": "*",
+            },
+        )
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)